etc/kayobe/environments/ci-multinode/networks.yml

```diff
@@ -81,7 +81,7 @@ internal_vlan: 101
 
 # External network
 external_cidr: 192.168.38.0/24
-external_mtu: 1450
+external_mtu: "{{ ansible_facts.default_ipv4.mtu - 50 }}"
```

Comment:
I'm sure we had this somewhere before?

Reply:
I thought we did as well but apparently not. Must've been left off a commit or something